python/cudf_polars/cudf_polars/testing/plugin.py

```diff
@@ -215,6 +214,7 @@ def pytest_configure(config: pytest.Config) -> None:
     # The test may segfault with the legacy streaming engine. We should
     # remove this skip when all polars tests use the new streaming engine.
     "tests/unit/streaming/test_streaming_group_by.py::test_streaming_group_by_literal[1]": "May segfault w/the legacy streaming engine",
+    "tests/unit/streaming/test_streaming_io.py::test_empty_sink_parquet_join_14863": "May segfault w/the legacy streaming engine",
```

Comment:
I need to investigate this failure a bit more. I don't think its related to the old streaming engine.